cmd/register/register_util.go

```diff
@@ -414,27 +441,107 @@ func getJSONSpec(message proto.Message) string {
 	return jsonSpec
 }
 
-func getFlyteTestManifest() ([]FlyteSnack, string, error) {
+func getFlyteTestManifest(org, repository string) ([]FlyteSnack, string, error) {
 	c := github.NewClient(nil)
 	opt := &github.ListOptions{Page: 1, PerPage: 1}
-	releases, _, err := c.Repositories.ListReleases(context.Background(), githubOrg, githubRepository, opt)
+	releases, _, err := c.Repositories.ListReleases(context.Background(), org, repository, opt)
 	if err != nil {
 		return nil, "", err
 	}
-	response, err := http.Get(fmt.Sprintf(flyteManifest, *releases[0].TagName))
-	if err != nil {
-		return nil, "", err
+	if len(releases) > 0 {
+		response, err := http.Get(fmt.Sprintf(flyteManifest, *releases[0].TagName))
+		if err != nil {
+			return nil, "", err
+		}
+		defer response.Body.Close()
+
+		data, err := ioutil.ReadAll(response.Body)
+		if err != nil {
+			return nil, "", err
+		}
+
+		err = json.Unmarshal(data, &FlyteSnacksRelease)
+		if err != nil {
+			return nil, "", err
+		}
+		return FlyteSnacksRelease, *releases[0].TagName, nil
 	}
-	defer response.Body.Close()
+	return nil, "", fmt.Errorf("Repository doesn't have any release")
+}
 
-	data, err := ioutil.ReadAll(response.Body)
+func getRemoteStoragePath(remoteLocation, file, identifier string) storage.DataReference {
+	return storage.DataReference(fmt.Sprintf("%v/%v-%v", remoteLocation, identifier, file))
+}
+
+func getAdditionalDistributionPath(remotePath string, conf *storage.Config) string {
+	if len(remotePath) == 0 {
```

Comment:
You should use [this](https://github.com/flyteorg/flytestdlib/blob/master/storage/storage.go#L48) to get the base path.

And you should use [this](https://github.com/flyteorg/flytestdlib/blob/master/storage/storage.go#L68) to join paths...